docs/Delivery-Practices/Design-and-Definition/3-Amigos-&-Readying-Backlog-Items/Design-Session-Guidelines.md

```diff
@@ -0,0 +1,65 @@
+- **Session Plan** - a plan of the design session, you should consult with the following to produce this; Principal Engineer, Software Architect, and VP of Engineering. 
+- **Existing Architecture**  - an understanding of existing platforms/infrastructure/design, this can be done in collaboration with others to produce this. 
+- **Investigations** - given the technical considerations, any components/tools must be investigated and a report produced.
+- **Session Length** - we should ensure the session length takes into account the size and complexity of the roadmap item (use your team to help choose a realistic time frame). 
+
+> :bulb: **Tip:** The person reasonable for the input isn't always the person generating the input, they just need to coordinate the generation of the inputs. Use the Empowerment Team and your sprint team to help generate the inputs.
+
+
+## During
+Here are some tips to ensure the success of the design session
+
+* As many key participants as possible in-person!
+* Interrogate solutions proposed to understand problems needing solving.
+* Be prepared to challenge ideas.
+* The session plan should be led by Tech Lead.
+
+## Outputs
+Once the session has ended there is an expectation we would have produced the following outputs:
+
+- **Low-fi delivery plan** - this should use ROM estimates only, the delivery plan should demonstrate how value will be delivered not when (e.g. critical path map)
+- **Raid Log** - this should be an ongoing output during the whole of the design session, consider having a dedicated raid logger to continually update the raid log. 
+- **Functional Design Definition** - this should detail how the solution will behave functionally.
+- **Technical Design** - this should detail how we will go about engineering the solution and should include any additional investigations needed to validate the technical design.
+- **High-level Test Plan** - this should detail the approach of how we will test the solution, this shouldn't be exhaustive but should contain the approach and any additional investigations needed.
+- **Design & Definition Plan** - outline the design & readying process remaining, our "Road to confidence" 
+
+###
+
+## Outcomes
+The following are the expected outcomes of a design session, if the team hasn't achieved the expected outcomes then it is an indication that further design work is necessary: 
+
+- Aligned on & confidence in functional & technical design
+- Aligned on a plan for design & definition
+- Understanding of unknown and investigations required
+- Stakeholders understand the route to problem resolution
+
+
+> :bulb: **Tip:** Given we are a department that works remotely when setting up an in-person design session, consider viability of addressing multiple problems. But be considerate of WIP and make sure that you have the ability to work on the outputs produced.
```

Comment:
Also consider whether the design session can align with existing commitments, e.g. before or after a team day or other in-person event that participants are already likely to be attending in-person.